rust/cubestore/src/metastore/mod.rs

```diff
@@ -1517,7 +1517,7 @@ impl RocksMetaStore {
         let mut opts = Options::default();
         opts.create_if_missing(true);
         opts.set_prefix_extractor(rocksdb::SliceTransform::create_fixed_prefix(13));
-        opts.set_merge_operator("meta_store merge", meta_store_merge, None);
+        opts.set_merge_operator("meta_store merge", meta_store_merge, meta_store_merge);
```

Comment:
NIT: let's use `set_merge_operator_associative`, that's what `rocksdb` uses in the docs

Reply:
Thanks!